configs/config.dell_optiplex_9010_sff

```diff
@@ -1,11 +1,17 @@
+CONFIG_LOCALVERSION="v0.1.0"
 CONFIG_USE_OPTION_TABLE=y
-CONFIG_USE_BLOBS=y
 CONFIG_VENDOR_DELL=y
-CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_FMDFILE="src/mainboard/dell/snb_ivb_workstations/default.fmd"
```

Comment:
It would be better to add this as default to mainboard Kconfig instead, something like:

```
config FMDFILE
       default "src/mainboard/\$(CONFIG_MAINBOARD_DIR)/default.fmd"
```

Reply:
I'd also check if this is needed at all, isn't the default FMD generated by the buildsystem enough? I don't see any nonstandard regions here: https://github.com/Dasharo/coreboot/blob/dasharo/src/mainboard/dell/snb_ivb_workstations/default.fmd

Reply:
[Done](https://github.com/Dasharo/coreboot/pull/583/commits/c7d25e78688438f79e4cff2e7b12849a643c0b7c)